test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -838,6 +838,27 @@ namespace PowershellTestConfigNamespace
             $result | Should -BeTrue
         }
     }
+
+    Describe "Validate Enable-PSSession Cmdlet" -Tags @("Feature", 'RequireAdminOnWindows') {
```

Comment:
There should be a test case where if you run `Enable-PSRemoting` or `Install-PowerShellRemoting.ps1` and either the PowerShell.6 endpoint or PowerShell.6.x.x endpoint exists, we don't overwrite it (modify some aspect of it's configuration like the ACL, run the cmdlet/script, verify the ACL wasn't reset)